components/popover/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -1,5 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Popover rtl render component should be rendered correctly in RTL direction 1`] = `<span />`;
```

Comment:
Seems strange it only render a empty span without rlt content.

Reply:
will be resolved after updating snapshots